### DISCUSSION TITLE
重新安装后出现了这个

### DISCUSSION THREAD

**Discussion by szy2333** [LocalizeLimbusCompany/LocalizeLimbusCompany#160] [常规 | General]
![YDMWW {TVZP5FCT964Y9V$R](https://github.com/LocalizeLimbusCompany/LocalizeLimbusCompany/assets/142065926/450e23a4-98b8-42ff-98f2-655c8e0a0175)

**Comment by N071** [comment]
这看起来仍然是旧版（MelonLoader）模组会出现的问题。您可以更新工具箱，使用最新版模组
